### PR TITLE
[DKP] Release Notes 2.2.2

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-90183

## Description of changes being made

Release notes for DKP release 2.2.2.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4567.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+layout: layout.pug
+navigationTitle: DKP 2.2.2 Release Notes
+title: DKP 2.2.2 Release Notes
+menuWeight: 20
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md:5]: So this file appears below 2.2.1 RN
```suggestion
menuWeight: 30
```

File: pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+### Fixed containerd vulnerability  (COPS-7531)
+
+DKP packages now include updated containerd and FIPS manifests, mitigating CVE-2022-23648, a known containerd vulnerability.
+
+### Dex CRDS are out of date (COPS-7341)
+
+The custom resource definitions used for configuring LDAP have been updated to include the correct syntax.
+
+### The Konvoy credentials plugin instructions embedded in the dex-k8s-authenticator dialogs don't work (COPS-7343)
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md:40]:
```suggestion
### The Konvoy credentials plugin instructions embedded in the dex-k8s-authenticator dialogs do not work (COPS-7343)
```

File: pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+| **Maximum**  | 1.22.x  |
+| **Default**  | 1.22.8  |
+
+## Fixes and Improvements
+
+### Fixed containerd vulnerability  (COPS-7531)
+
+DKP packages now include updated containerd and FIPS manifests, mitigating CVE-2022-23648, a known containerd vulnerability.
+
+### Dex CRDS are out of date (COPS-7341)
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md:36]:
```suggestion
### Outdated Dex CRDS (COPS-7341)
```

File: pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+### Fixed containerd vulnerability  (COPS-7531)
+
+DKP packages now include updated containerd and FIPS manifests, mitigating CVE-2022-23648, a known containerd vulnerability.
+
+### Outdated Dex CRDS (COPS-7341)
+
+The custom resource definitions used for configuring LDAP have been updated to include the correct syntax.
+
+### The Konvoy credentials plugin instructions embedded in the dex-k8s-authenticator containers do not work (COPS-7343)
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md:40]:
```suggestion
### The Konvoy credentials plugin instructions embedded in the dex-k8s-authenticator containers did not work (COPS-7343)
```

**Review by cneth — CHANGES_REQUESTED** [review]
I reviewed the konvoy2 and kommander change logs, we're missing release notes for the following fixes.  I've drafted a proposed RN but these NEED TO BE REVIEWED BY ENGINEERING...  

---
https://jira.d2iq.com/browse/D2IQ-90140.  (Konvoy - CAPZ updated).

RN:   Updated Azure support to fix an issue with specifying Additional Tags (COPS-7346)

The Cluster API Azure (CAPZ) support was updated to resolve a race condition that could prevent the creation of additional resource tags.

---
https://jira.d2iq.com/browse/D2IQ-86169 

RN:   Updated Traefik Forward Auth to support 'clusterStorage' option. (COPS-7344)

The Traefik Forward Auth (TFA) component was updated to support storing of session cookies in-cluster instead of in the browser.

---
https://jira.d2iq.com/browse/D2IQ-84000

RN:  Added Continuous Deployment(CD)/Gitops to the View role

The CD/Gitops resource was added to the 'View' clusterRole, so that users with only this permission can view the CD/Gitops resources.

**Comment by cneth** [comment]
Here's another one we found in our review:

https://jira.d2iq.com/browse/D2IQ-89452 (COPS-7336)

RN:   Kiali Platform Application not working properly (COPS-7336)

An installation problem that caused the Kiali platform application to malfunction has been corrected.

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+The mesosphere/dex-k8s-authenticator docker container now includes the appropriate binaries that allow users to download the referenced 'konvoy-async-plugin' after configuring a cluster using an external IDP for authentication.
+
+## Component updates
+
+When upgrading to this release, the following services and service components are upgraded to the listed version:
+
+| Common Application Name | APP ID                                         | Version | Component Versions                                                                                                                        |
+| ----------------------- | ---------------------------------------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| Cert Manager            | cert-manager                                   | 1.7.1   | - chart: 1.7.1<br>- cert-manager: 1.7.1                                                                                                 |
```
Comment (cmastr) [inline pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md:66]: Did these get updates from Brian Mendoza jira?

**Pull request merged**